doc/ref_arch/openstack/chapters/chapter02.rst

```diff
@@ -119,8 +121,8 @@ Table 2-1a: Reference Model Requirements: Cloud Infrastructure Software
 Profile Capabilities
 
    **1** Defined in the ``.bronze`` configuration in `RM section 4.2.6
-   Storage
-   Extensions <../../../ref_model/chapters/chapter04.md#426-storage-extensions>`__\ 
+   Storage Extensions
+   <../../../ref_model/chapters/chapter04.md#426-storage-extensions>`__
```

Comment:
@collivier Link not changed, it should be 
:ref:`ref_model/chapters/chapter04:storage extensions`